e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
   args:
     executable: /bin/bash
   register: result
-  until: "'No resources found.' in result.stdout"
+  until: "'No resources found.' in result.stderr"
```

Comment:
Would it be good to do a slightly stronger check with "result.rc == 0 & 'No resources found.' in result.stderr"